tests/Dockerfile

```diff
@@ -12,5 +12,6 @@ ADD . /app/
 WORKDIR /app/
 
 ENV BELUGA_TEST true
+ENV SECRET_BASE f5a9149cf77a47ddcc053a7731ec333a17b62238ff147345289a2d17fbd01eaec56946b5e339195484eb63140042e75c6ced01f46e196883eaee477240c46cdd
```

Comment:
🤔 is this a real secret? You can add env vars to our travis if you like.

Reply:
or just a dummy to get through the tests

Reply:
Just a dummy for tests, in real life it will generate one randomly at start